### ISSUE TITLE
Claims existing pages dont exist

### ISSUE THREAD

**Issue by lschwetlick** [lschwetlick/ADD-wiki#8]
Test Cases:
- sheet music
- eye movement

**Comment by lschwetlick** [comment]
was issue with capitalization. Solved

**Comment by lschwetlick** [comment]
Somehow this seems to be broken. 
On "Eye Movement" it only returns a result for "Eye movement" and "eye movement" but not for  "Eye Movement", and "eye Movement".
-> so youre thinking just turn all page titles into lower case right?
No, because on "United States Minor Outlying Islands" all words need to be capitalized for a result to be returned.

**Comment by chrisma** [comment]
Right, links extracted from Wikipedia pages pointing to other Wiki pages are correct and feature the correct capitalization (whatever that may be in each individual case).

The only issue exists with user supplied input. It might not be conform to the capitalization of the corresponding Wikipedia article. In fact, the user's search query might need more work than capitalization changes, e.g. in the case of misspellings or not hitting the exact title.

For this, Wikipedia (and a separate API endpoint) feature a search, which is able to map from user input to a (most likely) Wikipedia entry. This API needs to be called for user input and then the algorithm started with the returned wiki page.
